src/eq_schema/Block.test.js

```diff
@@ -34,4 +34,40 @@ describe("Block", () => {
       ]
     });
   });
+
+  describe("conversion of page types", () => {
+    it("should convert QuestonPage to Questionnaire", () => {
+      const authorJson = {
+        id: 1,
+        title: "Question 1",
+        description: "This is quesstion 1",
+        guidance: "",
+        pageType: "QuestionPage",
+        type: "General",
+        mandatory: false,
+        answers: []
+      };
+
+      const block = new Block(authorJson);
+
+      expect(block.type).toEqual("Questionnaire");
+    });
+
+    it("should convert InterstitialPage to Interstitial", () => {
+      const authorJson = {
+        id: 1,
+        title: "Question 1",
+        description: "This is quesstion 1",
+        guidance: "",
+        pageType: "InterstitialPage",
```

Comment:
If this was declared as above then you can just reassign the value:

`authorJson.pageType = "InterstitialPage"`